src/gui/dialogs/export_project.ui

```diff
@@ -81,25 +81,57 @@
          </widget>
         </item>
         <item>
+         <widget class="QWidget" name="depthWidget" native="true">
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>20</height>
+           </size>
+          </property>
+          <layout class="QVBoxLayout">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QLabel" name="labelBitDepth">
+             <property name="text">
+              <string>Depth:</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QComboBox" name="depthCB">
+             <item>
+              <property name="text">
+               <string>16 Bit Integer</string>
+              </property>
+             </item>
+             <item>
+              <property name="text">
+               <string>24 Bit Float</string>
```

Comment:
Commit 954659fa8e91fa42837ca213912d0be421faff5b changes the string from "24 Bit Float" to "24 Bit Integer". It seems that both options ("Integer" vs. "PCM") are used by other audio software, e.g. REAPER also uses "24 Bit PCM" and "32 Bit Float". However, in the end the naming is not that important to me and we shall just use one of the two standards. :)

You are also right that users who want to learn about the feature can just search for "bit depth".